src/EPR.Calculator.API/Builder/Summary/ThreeSa/ThreeSaUtil.cs

```diff
@@ -0,0 +1,108 @@
+﻿using EPR.Calculator.API.Builder.Summary.Common;
+using EPR.Calculator.API.Models;
+
+namespace EPR.Calculator.API.Builder.Summary.ThreeSA
+{
+    public static class ThreeSaUtil
+    {
+        public static decimal GetSAOperatingCostsScotlandTotalWithBadDebtProvisionSection3(Dictionary<MaterialDetail, CalcResultSummaryProducerDisposalFeesByMaterial> materialsCostSummary, Dictionary<MaterialDetail, CalcResultSummaryProducerCommsFeesCostByMaterial> costSummary, List<MaterialDetail> materials, CalcResult calcResult)
+        {
+            //1+paramOthers
+            Decimal OnePlusOtherParam = 1 + (calcResult.CalcResultParameterOtherCost.BadDebtValue / 100);
+
+            //1+4 apportenmnt
+
+            var oneplause = ConverttoDecimal(calcResult.CalcResultOnePlusFourApportionment.CalcResultOnePlusFourApportionmentDetails.Where(x => x.Name == "1 + 4 Apportionment %s").FirstOrDefault().ScotlandDisposalTotal);
+            var producerPercentage = GetTotal1Plus2ABadDebtPercentage(CalcResultSummaryUtil.GetTotalProducerDisposalFeeWithBadDebtProvision(materialsCostSummary), CalcResultSummaryUtil.GetTotalProducerCommsFeeWithBadDebtProvision(costSummary), materials, calcResult);
+            return GetSAOperatingCostsTotalWithoutBadDebtProvisionTitleSection3(calcResult) * ConverttoDecimal(producerPercentage.ToString()) * OnePlusOtherParam * oneplause;
+        }
+
+        public static decimal GetSAOperatingCostsWalesTotalWithBadDebtProvisionSection3(Dictionary<MaterialDetail, CalcResultSummaryProducerDisposalFeesByMaterial> materialsCostSummary, Dictionary<MaterialDetail, CalcResultSummaryProducerCommsFeesCostByMaterial> costSummary, List<MaterialDetail> materials, CalcResult calcResult)
+        {
+            //1+paramOthers
+            Decimal OnePlusOtherParam = 1 + (calcResult.CalcResultParameterOtherCost.BadDebtValue / 100);
+
+            //1+4 apportenmnt
+
+            var oneplause = ConverttoDecimal(calcResult.CalcResultOnePlusFourApportionment.CalcResultOnePlusFourApportionmentDetails.Where(x => x.Name == "1 + 4 Apportionment %s").FirstOrDefault().WalesDisposalTotal);
```

Comment:
Please change oneplause it to a relevant variable name. Also, please update other occurrences.